docs/_data/groups/2017.yml

```diff
@@ -103,6 +103,19 @@
     - name: Jeremy Dee
       email: message through Meetup
 
+- name: Minnesota PHP UG
+  location: Bloomington, MN, USA
+  website: https://www.meetup.com/mnphp/
+  meetup: https://www.meetup.com/mnphp/
```

Comment:
Hi @jsundquist - your site and meetup links don't resolve. Please check and correct when you get a chance. Thanks!

Reply:
Missing the hyphen. Will get it corrected.